### PR TITLE
add format dates in pantry #74

### PR DESCRIPTION
**Pull request by miguelparkour**

### CODE REVIEW

File: app/src/utils/index.tsx [hunk 1]
```diff
@@ -127,6 +127,20 @@ export const useGetExpiryObj = () => {
                 color: palette.warning.dark,
                 label: `${daysDiff} días`,
             };
+        } 
+        if (daysDiff >= 31 && daysDiff < 365) {
+            const monthsDiff = differenceInMonths(expiryDate,currentDate);
```
Comment (andyindahouse) [inline app/src/utils/index.tsx:132]: could you check what happen if there is 31 exact of difference in the function 'differenceInMonths'? Whih is the diff? 1 month? 0 month?

Comment (miguelparkour) [reply app/src/utils/index.tsx:132]: When the difference is 31 days, differenceInMonths returns 1 month, checked by console logs

Comment (andyindahouse) [reply app/src/utils/index.tsx:132]: perfect

**Review by andyindahouse — APPROVED** [review]
good job!

**Pull request merged**